docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -77,36 +77,36 @@ For subscription Office on Windows, the browser that is used is determined by th
 
 ## Working with Internet Explorer
 
-Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you have two options.
+Internet Explorer 11 doesn't support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you have two options.
 
 - Write your code in ECMAScript 2015 (also called ES6) or later JavaScript, or in TypeScript, and then compile your code to ES5 JavaScript using a compiler such as [babel](https://babeljs.io/) or [tsc](https://www.typescriptlang.org/index.html).
 - Write in ECMAScript 2015 or later JavaScript, but also load a [polyfill](https://en.wikipedia.org/wiki/Polyfill_(programming)) library such as [core-js](https://github.com/zloirock/core-js) that enables IE to run your code.
 
-For more information about these options, see [Support Internet Explorer 11](../develop/support-ie-11.md).
+For more information about these options, see [Support Internet Explorer 11 and Microsoft Edge Legacy](../develop/support-ie-11.md).
 
-Also, Internet Explorer 11 does not support some HTML5 features such as media, recording, and location. To learn more, see [Determine at runtime if the add-in is running in Internet Explorer](../develop/support-ie-11.md#determine-at-runtime-if-the-add-in-is-running-in-internet-explorer).
+Also, Internet Explorer 11 doesn't support some HTML5 features such as media, recording, and location. To learn more, see [Determine at runtime if the add-in is running in IE11 or Microsoft Edge Legacy](../develop/support-ie-11.md#determine-at-runtime-if-the-add-in-is-running-in-ie11-or-microsoft-edge-legacy).
 
 ## Troubleshoot Microsoft Edge issues
 
-### Service Workers are not working
+### Service Workers aren't working
 
-Office Add-ins do not support Service Workers when the original Microsoft Edge WebView, [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML), is used. They are supported with the [Chromium-based Edge WebView2](/microsoft-edge/hosting/webview2).
+Office Add-ins don't support Service Workers when Microsoft Edge Legacy's WebView, [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML), is used. They're supported with the [Chromium-based Edge WebView2](/microsoft-edge/hosting/webview2).
```

Comment:
Possessives on brand names sometimes don't translate well, so it's best to avoid them when possible.
```suggestion
Office Add-ins don't support Service Workers when the Microsoft Edge Legacy WebView, [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML), is used. They're supported with the [Chromium-based Edge WebView2](/microsoft-edge/hosting/webview2).
```